templates/homepage-blogs.tmpl

```diff
@@ -3,6 +3,7 @@
   {% for post in posts %}
     <div class="blog-content">
       <h2><a href="{{ post.permalink(lang) }}">{{ post.title(lang)|e }}</a></h2>
+      <div class="blog-author"><a href="/authors/{{ post.author() | slugify() }}">{{ post.author() }}</a></div>
```

Comment:
Maybe we could insert "By" in here so it is a bit more clear that is the author of the post.

```
<div class="blog-author">By<a href="/authors/{{ post.author() | slugify() }}">{{ post.author() }}</a></div>
```

I guess you might need to wrap that in a `p` tag though or something.

Reply:
I did try that and wasn't super-happy with how it looked. let me see if I can improve it.

Reply:
Ok, with a smaller font and the date as well, this looks better, see what you think.

Reply:
very nice!